src/Microsoft.NetCore.Analyzers/UnitTests/Data/ReviewSQLQueriesForSecurityVulnerabilitiesTests.cs

```diff
@@ -1580,20 +1586,20 @@ End Sub
         }
 
         [Fact]
-        public void MissingWellKnownTypes_NoDiagnostic()
+        public async Task MissingWellKnownTypes_NoDiagnostic()
         {
-            VerifyCSharp(@"
-class C { }", ReferenceFlags.RemoveSystemData);
+            await VerifyCS.VerifyAnalyzerAsync(@"
+class C { }");
 
-            VerifyBasic(@"
-Class C
-End Class", ReferenceFlags.RemoveSystemData);
+            await VerifyVB.VerifyAnalyzerAsync(@"
+            Class C
+            End Class");
```

Comment:
📝 These lines are part of a verbatim string literal, so the indentation should not change